templates/dag-deploy/dag-server-statefulset.yaml

```diff
@@ -53,6 +53,8 @@ spec:
           resources:
             {{- toYaml .Values.dagDeploy.resources | nindent 12 }}
           env:
+          - name: HOUSTON_SERVICE_ENDPOINT
+            value: "https://{{ .Values.platform.release }}-houston.{{ .Values.platform.namespace }}.svc.cluster.local.:8871/v1/"
```

Comment:
https wont work and it will fail

Reply:
@pgvishnuram can you elaborate?
`https wont work and it will fail`

Reply:
@rishkarajgi houston TLS is terminated at nginx when hit on the external URL. internally the houston service is only provided over HTTP.